charts/cni-metrics-helper/README.md

```diff
@@ -0,0 +1,64 @@
+# CNI METRICS HELPER
+
+This chart provides a Kubernetes deployment for the Amazon VPC CNI Metrics Helper, which is used to collect metrics for the Amazon VPC CNI plugin for Kubernetes.
+
+## Prerequisites
+
+- Kubernetes 1.11+ running on AWS
+- Helm 3.0+
+
+## Installing the Chart
+
+Clone the Amazon VPC CNI for Kubernetes repository to your local machine.
+
+```shell
+$ git clone https://github.com/aws/amazon-vpc-cni-k8s.git
+```
+Use the helm install command to install the chart into your Kubernetes cluster
+
+```shell
+$ helm install cni-metrics-helper ./amazon-vpc-cni-k8s/charts/cni-metrics-helper
+```
+
+## Configuration
+
+The following table lists the configurable parameters for this chart and their default values.
+
+| Parameter                  | Description                                                   | Default            |
+|----------------------------|---------------------------------------------------------------|--------------------|
+| fullnameOverride           | Override the fullname of the chart                            | cni-metrics-helper |
+| image.region               | ECR repository region to use. Should match your cluster       | us-west-2          |
+| image.tag                  | Image tag                                                     | v1.11.4            |
+| image.account              | ECR repository account number                                 | 602401143452       |
+| image.domain               | ECR repository domain                                         | amazonaws.com      |
+| env.USE_CLOUDWATCH         | Whether to export CNI metrics to CloudWatch                   | true               |
+| env.AWS_CLUSTER_ID         | ID of the cluster to use when exporting metrics to CloudWatch | default            |
+| metricUpdateInterval       | Interval at which to update CloudWatch metrics, in seconds    | 60                 |
```

Comment:
nit: `env.METRIC_UPDATE_INTERVAL`, and the default is 30

I would update the description (will take multiple lines) to:
```
Interval at which to update CloudWatch metrics, in seconds
Metrics are published to CloudWatch at 2*METRIC_UPDATE_INTERVAL
```

Reply:
Done!